src/components/button/Button.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import { Button } from '@components/index';
+
+test('Button renders with correct text', () => {
+  const handleClick = vi.fn();
+  render(<Button text="Click me" onClick={handleClick} />);
```

Comment:
And then you can add text as a child and add other builtin Button attributes:

```suggestion
  render(<Button onClick={handleClick}>Click me</Button>);
```